crates/ruff_python_formatter/src/comments/visitor.rs

```diff
@@ -22,30 +22,38 @@ use crate::comments::placement::place_comment;
 use crate::comments::{CommentLinePosition, CommentsMap, SourceComment};
 
 /// Visitor extracting the comments from an AST.
-#[derive(Debug, Clone)]
-pub(crate) struct CommentsVisitor<'a> {
-    builder: CommentsBuilder<'a>,
+#[derive(Debug)]
+pub(crate) struct CommentsVisitor<'a, 'b, Collector>
```

Comment:
We can make `CommentsVisitor` `pub(super)` instead of increasing the visibility of everything else.